daemon/src/lib.rs

```diff
@@ -70,6 +71,10 @@ pub mod wire;
 /// determine whether the maker is online.
 pub const HEARTBEAT_INTERVAL: Duration = Duration::from_secs(5);
 
+/// Duration between the restart attempts after a supervised actor has quit with
+/// a failure.
+pub const RESTART_INTERVAL: Duration = Duration::from_secs(5);
```

Comment:
:question: Does this apply to all supervised actors in this file? The name of this constant and the commit message are not completely aligned imo.

Reply:
it applies to all actors that use the constant; any suggestions to improve it?